src/engraving/dom/excerpt.cpp

```diff
@@ -1153,6 +1153,8 @@ void Excerpt::cloneStaves(Score* sourceScore, Score* dstScore, const std::vector
         measures->add(newMeasure);
     }
 
+    measures->updateTickIndex();
```

Comment:
What's the purpose of this? I thought push_back(), add(), etc. already update the map as needed, right?